src/main/java/model/Deck.java

```diff
@@ -36,6 +36,28 @@ public Deck(String deckName, TagUUID tagUUID, CardUUID cardUUID) {
     public String getDeckName() {
         return deckName;
     }
+    public boolean cardInDeck(CardUUID cardUUID) {
```

Comment:
Function name has same issue.

Reply:
Issues resolved in edab2bc2b42c131e9ab63358f987ffae5538ebed